panels/updates/cc-tariff-editor.c

```diff
@@ -301,8 +301,8 @@ update_adjustments (CcTariffEditor *self)
 
   g_debug ("Updating adjustments");
 
-  start = g_date_time_new_from_unix_utc (self->seconds_to_start);
-  end = g_date_time_new_from_unix_utc (self->seconds_to_end);
+  start = g_date_time_new_from_unix_local (self->seconds_to_start);
+  end = g_date_time_new_from_unix_local (self->seconds_to_end);
```

Comment:
AFAIU, this is not related to 12h systems only but affects both formats, right? So it should be done in a separate commit to avoid confusion.

Reply:
No, it is related to 12h. And I think it shouldn't be split in 2 commits, because it'll be squashed in the next rebase and nothing from it will be preserved, not even the commit message.

Reply:
Just FTR, as discussed on Slack, it **does** affect both formats. But doing it as a fixup commit makes this less important.